### DISCUSSION TITLE
Font size?

### DISCUSSION THREAD

**Discussion by hkarl** [mortbopet/Ripes#110] [Q&A]
Hi, 
silly question, but is it possibly to increase the font size in the various views (especially, editor, gpr, memory)? I am considering Ripes for teaching, and I doubt anybody would be able to read that on a projector in a bigger lecture hall. 

Thanks
Holger

**Comment by mortbopet** [comment]
Hi @hkarl !
You can increase the font size of the editor by `ctrl+scroll`'ing while hovering your mouse over it. This is the same for the datapath and cache views.
There's currently no way to specify the font size of the memory view. In general, if you'd like _everything_ to be scaled up, i think your best bet is to reduce the resolution of your monitor.

**Reply by hkarl** [reply]
Ah, cmd+scroll on Mac does work (odd, I thought I had tried that). Thanks. 

I'll see how it works with different resolutions, thanks. Great project, btw!
